stylesheets/rules.css

```diff
@@ -0,0 +1,4 @@
+.rules-container{
+    border: 0.12em solid var(--action-color);
```

Comment:
why ems ?

Reply:
typo! 😅
Thanks for pointing it out. Will change it.